docs/rules/README.md

```diff
@@ -229,6 +228,7 @@ These rules are only relevant to ES6 environments.
 * [no-const-assign](no-const-assign.md) - disallow modifying variables that are declared using `const` (recommended)
 * [no-dupe-class-members](no-dupe-class-members.md) - disallow duplicate name in class members (recommended)
 * [no-new-symbol](no-new-symbol.md) - disallow use of the `new` operator with the `Symbol` object (recommended)
+* [no-restricted-imports](no-restricted-imports.md) - restrict usage of specified ES6 imports
```

Comment:
I'm not sure how they're not consistent with the main heading, but you mention not to worry about that, so I won't. :)

Also, hopefully before you guys make further changes to the organization of the README, I'm just submitting these minor (and perhaps less controversial) fixes first, since I also have a PR ready which I plan to submit which gets rid of the environment sections entirely (putting the environment info in parentheses to allow users of the README to proceed through the page on a sequence based primarily on the cost-benefit degree).